main.py

```
@@ -636,34 +640,30 @@ def create_reading_order(self):
+        self.order_display_.addWidget(self.order_editor)
         self.stack.setCurrentWidget(self.order_widget)
 
-    def open_order_editor(self, id: int, name: str):
-        self.order_editor = ReadingOrderEditor(id, name)
-        self.order_editor.setWindowFlag(Qt.WindowType.Window)
-        # order_editor.show()
-        self.order_editor.showFullScreen()
+    def order_search(self, text: str):
+        display_info = text_search(text)
+        self.order_editor.library_panel.set_comics(display_info or [])
+        return
```

Ruff (0.15.12)

[warning] Useless `return` statement at end of function

Remove useless `return` statement

(PLR1711)